parser.y

```diff
@@ -5789,6 +5788,37 @@ TableNameList:
 		$$ = append($1.([]*ast.TableName), $3.(*ast.TableName))
 	}
 
+TableNameOptWild:
+	Identifier OptWild
+	{
+		$$ = &ast.TableName{Name:model.NewCIStr($1)}
+	}
+|	Identifier '.' Identifier OptWild
+	{
+		$$ = &ast.TableName{Schema:model.NewCIStr($1),	Name:model.NewCIStr($3)}
+	}
+
+TableAliasRefList:
+	TableNameOptWild
+	{
+		tbl := []*ast.TableName{$1.(*ast.TableName)}
+		$$ = tbl
+	}
+|	TableAliasRefList ',' TableNameOptWild
+	{
+		$$ = append($1.([]*ast.TableName), $3.(*ast.TableName))
+	}
+
+OptWild:
+	%prec empty
+	{
+		$$ = nil
+	}
+|	'.' '*'
+	{
+		$$ = nil
+	}
```

Comment:
No need to write `$$ = nil` for these two branches.

Reply:
ok